### PR TITLE
Checkpoint 2: User sign up

### PR DESCRIPTION
**Pull request by MelissaAJohnson**
Please confirm gemfile looks correct.
Sign Up page needs styling love and home (user not logged in) page needs a tweaking.

Home
root

SignUp
sign up

Signed In User Nav
screen shot 2016-08-11 at 8 15 26 pm

### CODE REVIEW

File: app/assets/stylesheets/home.scss [hunk 1]
```diff
@@ -0,0 +1,3 @@
+// Place all the styles related to the home controller here.
```
Comment (brittanyjspringer) [inline app/assets/stylesheets/home.scss]: Please delete this file.

File: app/assets/javascripts/home.coffee [hunk 2]
```diff
@@ -0,0 +1,3 @@
+# Place all the behaviors and hooks related to the matching controller here.
```
Comment (brittanyjspringer) [inline app/assets/javascripts/home.coffee]: Please delete this file.

File: app/helpers/home_helper.rb [hunk 3]
```diff
@@ -0,0 +1,2 @@
+module HomeHelper
```
Comment (brittanyjspringer) [inline app/helpers/home_helper.rb]: Please delete this file.

**Comment by brittanyjspringer** [comment]
@melissajstudent just a couple of unused files marked for you to delete. Ping me here when you have it!

**Comment by brittanyjspringer** [comment]
Also, I assigned myself on the right side. Please assign me going forward 💃

**Comment by MelissaAJohnson** [comment]
@wonderwoman13, unnecessary files removed and new commit made. Is the new commit automatically added to the pull request?

**Comment by brittanyjspringer** [comment]
@melissajstudent it is! I already merged the PR 👍

**Pull request merged**